app/scripts/models/reliers/fx-desktop.js

```diff
@@ -78,6 +80,10 @@ define([
      */
     isCustomizeSyncChecked: function () {
       return !! (this.isSync() && this.get('customizeSync'));
+    },
+
+    isSignupDisabled: function () {
+      return this.get('exclude_signup') === '1';
     }
```

Comment:
I'm also partial to just ripping it out, if `exclude_signup` was meant to only be a temporary hack.